src/html/jbuild

```diff
@@ -3,4 +3,4 @@
 (library
  ((name html)
   (preprocess (pps (bisect_ppx -conditional)))
-  (libraries (model tyxml))))
+  (libraries (model tyxml reason))))
```

Comment:
Do we still have a dep on `reason` here? I think the code that uses `reason` is commented out, but not sure if I missed some.